### DISCUSSION TITLE
Can't set tunnel lights

### DISCUSSION THREAD

**Discussion by luna-sayles** [carla-simulator/carla#4389] [Q&A] [answered]
Hi, I was experimenting with Town03 and I noticed that I can't seem to edit the lights in the tunnel even when every other light changes to my specifications. Are these lights not actual light objects? Or is there something special needed to interact with them? Any help would be great

**Comment by Madecu** [comment, marked as answer]
Hi @Athena-Pyre ,
You are right, those lights can't be controlled through API. We will fix that in the next release. We will add the component they need, so you don't have to go inside unreal engine's project to edit them.
